apps/app/src/pages/Build/ScreenshotDiffStatusIcons.js

```diff
@@ -3,12 +3,15 @@ import {
   ChecklistIcon,
   FileAddedIcon,
   FileRemovedIcon,
+  FileIcon,
 } from "@primer/octicons-react";
 
 export function ScreenshotDiffStatusIcon(type) {
   switch (type) {
     case "added":
       return FileAddedIcon;
+    case "failed":
+      return FileIcon;
```

Comment:
Weird.

Reply:
I didn't find a more suitable "file" icon in Octicons lib. 
In an other hand we can give up "file icons" and use icons like "x-circle-fill"

https://primer.style/octicons/x-circle-fill-24